### DISCUSSION TITLE
MacOS 开启TUN模式后被劫持退出Clash也无法正常使用国内网络

### DISCUSSION THREAD

**Discussion by sunjianan9900** [libnyanpasu/clash-nyanpasu#1802] [Q&A]
不要删帖，因为我确实没有搜索到解决方案
<img width="798" alt="image" src="https://github.com/user-attachments/assets/13ec9206-ac9d-4389-8d5d-77acdc21f1bb">
TUN模式开启时，MAC使用正常，可以访问谷歌和百度
TUN模式关闭时，不正常，可以访问谷歌，但是无法访问百度
TUN模式关闭并退出Clash时，不正常，谷歌百度均无法访问

目前TUN模式似乎对我的电脑做了什么配置，而且关闭也不会自动还原
尝试了退出后清理DNS脚本命令“sudo killall -HUP mDNSResponder”，使用cleanmymac清理DNS缓存，都不可以

**Comment by keiko233** [comment]
我这边暂时没有遇到过，检查dns可以用nslookup查看
